simple/stats/runner.py

```diff
@@ -102,6 +103,9 @@ def run(self):
           list(self.nodes.variables.values()),
           self.nl_dir_fh.make_file(constants.SENTENCES_FILE_NAME))
 
+      # Write import info to DB.
+      self.db.insert_import_info(status=ImportStatus.SUCCESS)
```

Comment:
should we worry about exception when writing the import info?

Reply:
With any exception, the transaction will not be committed so we are ok.

In a later PR, I want to write the import info in a separate transaction so we can record failed imports as well.